docs/messaging/messaging-spans.md

```diff
@@ -216,8 +217,8 @@ SHOULD be set according to the following table, based on the operation a span de
 
 | Operation name | Span kind|
 |----------------|-------------|
-| `publish`      | `PRODUCER` if the context of the "Publish" span is used as creation context. |
 | `create`       | `PRODUCER` |
+| `publish`      | `PRODUCER` if the context of the "Publish" span is used as creation context. |
```

Comment:
If it's not a `PRODUCER` span, we shouldn't make any assumptions about the span kind. It could be a `CLIENT` span, however, there could also be transport-level `CLIENT` spans, in which case the "publish" span would be `INTERNAL`.

Reply:
Ok sounds good.